Mathlib/Algebra/BigOperators/List/Lemmas.lean

```diff
@@ -185,7 +185,7 @@ theorem unop_map_list_prod {F : Type*} [FunLike F M Nᵐᵒᵖ] [MonoidHomClass
 namespace MonoidHom
 
 /-- A morphism into the opposite monoid acts on the product by acting on the reversed elements. -/
-@[deprecated _root_.unop_map_list_prod]
+@[deprecated (since := "2023-01-10")] -- _root_.unop_map_list_prod
```

Comment:
Are you dropping the deprecated name here? If I have to choose, I would prefer to keep the name and put the date in the comment, as you do elsewhere.

Reply:
Sure, addressed.